### ISSUE TITLE
tcpdump captured data without TLS, when I use eggroll with TLS in docker compose mode.

### ISSUE THREAD

**Issue by hqsh** [FederatedAI/KubeFATE#742]
**What deployment mode you are use?  **
docker-compose

**What KubeFATE and FATE version you are using?  **
FATE 1.7.1.1

**What OS you are using for docker-compse or Kubernetes? Please also clear the version of OS.  **
CentOS Linux release 7.6.1810 (Core)
Docker version 20.10.12, build e91ed57
Docker Compose version v2.2.3

**To Reproduce**
I follow the document (https://github.com/FederatedAI/KubeFATE/blob/develop-1.9.0/docs/Eggroll_with_TLS.md#eggroll-with-tls) to make fate eggroll run with TLS. And I found rollsite docker logs have "gRPC server at port=9380 starting in secure mode. server private key path: /data/projects/fate/eggroll/conf/cert/server.key, key crt path: /data/projects/fate/eggroll/conf/cert/server.crt, ca crt path: /data/projects/fate/eggroll/conf/cert/ca.pem" and "secure server started at 9380".

I run LR model (1 guest / 1 host / 1 arbiter) and use the command: "sudo tcpdump -i eth0 host [host ip] or [guest ip] -w out.cap" on arbiter server to capture the data from guest and host to arbiter.

**What happen?**
I found the captured data is without TLS. They are TCP data which is same as deployment without TLS. Because I add logs in the fate LR model code to compare the original pilliar number pickle data. It is found in tcpdump data. （我发现抓包到的数据是没有 TLS 加密的，只是普通的 TCP 数据。因为我在 fate 的 LR 模型 guest、host 往 arbiter 传 pilliar 加密的 ndarray 数据（pickle 对应的二进制）的地方加了日志，结果发现发送的数据在 tcpdump 中能完全找到，这和我没有使用 TLS 加密部署时候抓包比对日志的情况一样）

I wonder where is not correct?

**Comment by JingChen23** [comment]
I'm not sure about that neither.
Please report the issue on:
https://github.com/WeBankFinTech/eggroll

**Comment by hqsh** [comment]
> I'm not sure about that neither. Please report the issue on: https://github.com/WeBankFinTech/eggroll

Thanks for your advice.

I am sure I have followed your document, and have checked serveral times. I  have reported the issue on eggroll.

However, as the issue I have reported on eggroll or fate long ago, it hasn't any reply yet.

So can you help to notice eggroll team to read the issue?

https://github.com/WeBankFinTech/eggroll/issues/588

Thank you!

**Comment by hqsh** [comment]
I have found more useful detail.

I read the source code of eggroll. And I found eggroll (rollsite) start insecure channel at 9370, and if configured ssl files, it start secure channel at 9380 additionally. It is same as the log in rollsite.

Howerer, kubefate docker compose only open 9370 on rollsite container, but open 9380 on python (fateflow server) container.

$ docker ps
CONTAINER ID   IMAGE                                                  COMMAND                  CREATED        STATUS        PORTS                                                                                NAMES
6eb8201c3409   redis:5                                                "docker-entrypoint.s…"   15 hours ago   Up 15 hours   6379/tcp                                                                             serving-10001-redis-1
32bbbc1cbe77   federatedai/serving-admin:2.0.4-release                "/bin/sh -c 'java -c…"   15 hours ago   Up 15 hours   8000/tcp, 0.0.0.0:8350->8350/tcp                                                     serving-10001-serving-admin-1
4b28830753dc   bitnami/zookeeper:3.7.0                                "/opt/bitnami/script…"   15 hours ago   Up 15 hours   0.0.0.0:2181->2181/tcp, 8080/tcp, 0.0.0.0:49496->2888/tcp, 0.0.0.0:49495->3888/tcp   serving-10001-serving-zookeeper-1
a9a9bc9b7332   federatedai/serving-server:2.0.4-release               "/bin/sh -c 'java -c…"   15 hours ago   Up 15 hours   0.0.0.0:8000->8000/tcp                                                               serving-10001-serving-server-1
ea615a352fe8   federatedai/serving-proxy:2.0.4-release                "/bin/sh -c 'java -D…"   15 hours ago   Up 15 hours   0.0.0.0:8059->8059/tcp, 0.0.0.0:8869->8869/tcp, 8879/tcp                             serving-10001-serving-proxy-1
146d38fdefc3   federatedai/client:1.7.1.1-release                     "/bin/sh -c 'flow in…"   15 hours ago   Up 15 hours                                                                                        confs-10001-client-1
760ad1c4bb0e   federatedai/fateboard:1.7.1.1-release                  "/bin/sh -c 'java -D…"   15 hours ago   Up 15 hours   0.0.0.0:8080->8080/tcp                                                               confs-10001-fateboard-1
2844309f055d   federatedai/python:1.7.1.1-release                     "container-entrypoin…"   15 hours ago   Up 15 hours   0.0.0.0:9360->9360/tcp, 8080/tcp, 0.0.0.0:9380->9380/tcp                             confs-10001-python-1
36aeeb09583f   federatedai/eggroll:1.7.1.1-release                    "/tini -- bash -c 'j…"   15 hours ago   Up 15 hours   4670/tcp, 8080/tcp                                                                   confs-10001-clustermanager-1
a5f277f1dc1d   federatedai/eggroll:1.7.1.1-release                    "/tini -- bash -c 'j…"   15 hours ago   Up 15 hours   4671/tcp, 8080/tcp                                                                   confs-10001-nodemanager-1
c73b7e1e034c   federatedai/eggroll:1.7.1.1-release                    "/tini -- bash -c 'j…"   15 hours ago   Up 15 hours   8080/tcp, 0.0.0.0:9370->9370/tcp                                                     confs-10001-rollsite-1
d6cddb9513b6   mysql:8.0.28                                           "docker-entrypoint.s…"   15 hours ago   Up 15 hours   3306/tcp, 33060/tcp                                                                  confs-10001-mysql-1

So, maybe it is not the issue for eggroll. Is it the issue needs colaboration of kubefate and eggroll?

I have tried to modify the 9370 and 9380 port in eggroll.properties and training_template/docker-compose-eggroll.yml. But the job cannot be submitted to fateflow server now.

Thank you.
